workshop/settings.py

```diff
@@ -38,9 +38,18 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'drf_yasg',
     'v1',
 ]
 
+SWAGGER_SETTINGS = {
```

Comment:
Replace with JWT authentication.

Reply:
@Veskor since JWT authentication should be used instead of basic one could we implement `drf-spectacular` (i.e. replace `drf_yasg` with it)?
More info about it can be found [here](https://stackoverflow.com/a/72149357/6805866).

Reply:
That's a good idea, shouldn't be too hard!

Reply:
@Veskor `drf-spectacular` implemented!